s3tests/functional/__init__.py

```diff
@@ -110,12 +109,12 @@ def nuke_prefixed_buckets_on_conn(prefix, name, conn):
 def nuke_prefixed_buckets(prefix):
     # If no regions are specified, use the simple method
     if targets.main.master == None:
-        for name, conn in s3.items():
+        for name, conn in list(s3.items()):
```

Comment:
Does this work without the list(s3.items()) call, i assume dict.items() generator should be iterable without needing to realize the list?